lib/routes/dataguidance/index.ts

```diff
@@ -0,0 +1,65 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+
+export const route: Route = {
+    name: 'Data Guidance News',
```

Comment:
```suggestion
    name: 'News',
```
Don't repeat namespace's `name`